en/includes/guides/user-self-service/self-register.md

```diff
@@ -55,6 +55,17 @@ The user can follow the steps below to sign up using an email.
 
 3. Click **Sign Up** to create the account.
 
-4. If **Account Verification** is enabled, the user can confirm the account from the email.
+4. If **Account Verification** enabled, the user can confirm the account from the email.
 
-    ![Account verification email notification]({{base_path}}/assets/img/guides/organization/self-service/customer/account-verification-email.png){: width="500" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
\ No newline at end of file
+    ![Account verification email notification]({{base_path}}/assets/img/guides/organization/self-service/customer/account-verification-email.png){: width="500" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+{% if product_name == "Asgardeo" or (product_name == "WSO2 Identity Server" and is_version > "7.1.0" ) %}
+5. If **Account Verification** enabled, users can resend the account verification email if needed.
+    - If **Activate Account Immediately** enabled, after successfully logging into My Account, a warning message with a resend link will appear. Click this link to resend the account verification email.
```

Comment:
```suggestion
    - If **Activate Account Immediately** enabled, after successfully logging into **My Account**, a warning message with a resend link will appear. Click this link to resend the account verification email.
```

Reply:
Resolved via https://github.com/wso2/docs-is/pull/5355/commits/8aab8b464237b6ab3565afe5ecc80b52e7d3197a